owner/manager_test.go

```diff
@@ -80,15 +80,15 @@ func TestSingle(t *testing.T) {
 	)
 	err = d.Start(nil)
 	if err != nil {
-		t.Fatalf("DDL start failed %v", err)
+		require.NoErrorf(t, err, "DDL start failed %v")
```

Comment:
`NoErrorf` is like `Noerror(testing, error)`, but with formats `Noerror(testing, error, msg, args...)`. You can see it as a combination of `NoError(testing, error)` + `Printf(msg, args...)`. So should be `require.NoErrorf(t, err, "DDL start failed %v", err)`.

But, since the require assertion will print `err` variable/the assertion variable anyway, you can just write  `require.NoErrorf(t, err, "DDL start failed")`.

Same to your other format functions.